.github/labeler.yml

```
@@ -0,0 +1,18 @@
+core:
+  - quantum/**/*
+  - tmk_core/**/*
+  - drivers/**/*
+  - tests/**/*
+  - util/**/*
```

In prep for `develop`'s merge, should we get this in?
```suggestion
  - util/**/*
  - platforms/**/*
```